src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskActionsControllerTest.java

```diff
@@ -427,6 +435,46 @@ void should_return_task_roles_with_permissions() {
         assertEquals(response.getBody().getPermissionsList(), taskRolePermissions);
     }
 
+
+    @Test
+    void should_return_201_response_for_tasks_deletion() {
```

Comment:
May be need to change the signature to 200 than 201 ?

Reply:
Please refer to my previous comment